src/python/cdr.py

```diff
@@ -0,0 +1,82 @@
+
+# coding: utf-8
+import pandas as pd
+import json
+import numpy
+import math
+from pandas.io.json import json_normalize
+
+
+def df_to_geojson(df, properties):
+    """
+    A function that changes Pandas dataFrame to Geojson.
+    
+    Parameters
+    ----------
+    df : DataFrame
+       This is a joined dataframe between CDR data and a grid data.
+    properties : list
+       Column names of CDR properties.
+
+    Returns
+    -------
+    geojson : dict
+
+
+    """
+    geojson = {'crs': {'type': 'name', 
+                       'properties': {"name": "urn:ogc:def:crs:EPSG::4326"}}, 
+               'type': 'FeatureCollection', 'features': []}
+    for _, row in df.iterrows():
+        feature = {'type': 'Feature', 'properties': {}, 
+            'id': row['id'],
+            'geometry': {'type': 'Polygon', 'coordinates': []}}
+        feature['geometry']['coordinates'] = row['geometry.coordinates']
+        for prop in properties:
+            if math.isnan(row[prop]):
+                feature['properties'][prop] = 0
+            else:
+                feature['properties'][prop] = row[prop]
+        geojson['features'].append(feature)
+    return geojson
+
+
+def join_cdr_grid(cdr, grid):
+    """
+    A function that outputs a joined table (dict) from CDR and GRID data.
+    
+    Parameters
+    ----------
+    cdr : pandas DataFrame
+       CDR data loaded in a pandas DataFrame.
+    grid : pandas Series
+       A pandas Series data of a city grid.
+
+    Returns
+    -------
+    geojson : dict
+
+
+    """
+    
+
+    cdr.columns = ["cellId", "time", "countryCode", "smsIn", "smsOut",
```

Comment:
Should we make this an input to the function? In case we want to reuse this pattern, or use it with some other column names?